### ISSUE TITLE
Cohort year adder shows "(undefined)" after Raikes members' names beyond Cohort 2012

### ISSUE THREAD

**Issue by amiinoo** [CodyJung/itcheck#8]
issue is as described in title. as a member of cohort 2019, the cohort year adder only seems to work on people in cohort 2012 and below. I don't know very much JS, but it's most likely an issue with the cohort-year-adder.js script. mind checking it out for me?

**Comment by CodyJung** [comment]
Yeah, I can get it updated when I do the next build. I just don't have the username-year combinations.
